### DISCUSSION TITLE
ND-NotAwardedReasonUnpublish

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#462] [Q&A] [answered]
Good Morning Lab !
I have some difficulties to make xml notice for ND-NotAwardedReasonUnpublish.
The reason, I think , is that , perhaps, it lacks a node.

You put those fields as children  of ND-NotAwardedReasonUnpublish : 
BT-195(BT-144)-LotResult[repeatable=false,xpathRelative=efbc:FieldIdentifierCode]
BT-196(BT-144)-LotResult[repeatable=false,xpathRelative=efbc:ReasonDescription]
BT-197(BT-144)-LotResult[repeatable=false,xpathRelative=cbc:ReasonCode]
BT-198(BT-144)-LotResult[repeatable=false,xpathRelative=efbc:PublicationDate]

in XSD, those fields are children of efac:FieldsPrivacy 

<xsd:complexType name="FieldsPrivacyType">
		<xsd:sequence>
			<xsd:element ref="efbc:FieldIdentifierCode" minOccurs="1" maxOccurs="unbounded"/>
			<xsd:element ref="cbc:ReasonCode" minOccurs="1" maxOccurs="1"/>
			<xsd:element ref="efbc:ReasonDescription" minOccurs="0" maxOccurs="unbounded"/>
			<xsd:element ref="efbc:PublicationDate" minOccurs="1" maxOccurs="1"/>
		</xsd:sequence>
	</xsd:complexType>

so in the xpathRelative of ND-NotAwardedReasonUnpublish : efac:DecisionReason/efac:FieldsPrivacy ( without predicate) there are
2 xsd node : 

first 
<xsd:complexType name="DecisionReasonType">
		<xsd:sequence>
			<xsd:element ref="efac:FieldsPrivacy" minOccurs="0" maxOccurs="unbounded"/>
			<xsd:element ref="efbc:DecisionReasonCode" minOccurs="1" maxOccurs="1"/>
		</xsd:sequence>
	</xsd:complexType>

and second <xsd:complexType name="FieldsPrivacyType">

so I think ,you should put the 4 fields as children of  a node, and set this node child of ND-NotAwardedReasonUnpublish
What do you think about it ?
Thanks and Regards !
Renato

**Comment by YvesJo** [comment, marked as answer]
Thanks Renato, I'll look into it.

**Reply by YvesJo** [reply]
I had a look back until SDK 1.4 and could just find ND-NotAwardedReasonUnpublish as parent node of the 4 fields. Is the issue related to the NTDs instead?
